client/src/RatingsReviews/lib/validateEmail.js

```diff
@@ -0,0 +1,17 @@
+function validateEmail(string) {
+  const regex = new RegExp('[a-z0-9]+@[a-z]+\.[a-z]{2,3}');
+  if (regex.test(string)) {
+    return true;
+    console.log('good email')
```

Comment:
might want to delete the console.logs for final code!

Reply:
good catch!